larpandoracontent/LArHelpers/LArMCParticleHelper.h

```diff
@@ -392,6 +392,14 @@ class LArMCParticleHelper
      */
     static bool IsDescendentOf(const pandora::MCParticle *const pMCParticle, const int pdg, const bool isChargeSensitive = false);
 
+    /**
+     *  @brief  Retrieve a linearised representation of the MC particle hierarchy in breadth first order
+     *
```

Comment:
Would be great if can set a precedent for adding a bit more info here, and confirm there's nothing already present that is very similar.